components/scream/src/physics/p3/p3_functions_ice_melting_impl.hpp

```diff
@@ -1,5 +1,5 @@
 #ifndef P3_FUNCTIONS_ICE_MELTING_IMPL_HPP
-#define P3_FUNCTIONS_ICE_MELTING_IMPL_HPP
+//#define P3_FUNCTIONS_ICE_MELTING_IMPL_HPP
```

Comment:
@PeterCaldwell : What is the reason for commenting out this line?

Reply:
Good catch @singhbalwinder - I have no idea why this was commented out.